zboxcore/sdk/copyworker.go

```diff
@@ -190,12 +191,14 @@ func (req *CopyRequest) ProcessCopy() error {
 	wg.Add(activeBlobbers)
 	commitReqs := make([]*CommitRequest, activeBlobbers)
 
+	var uid uuid.UUID
 	var c int
 	for i := req.copyMask; !i.Equals64(0); i = i.And(zboxutil.NewUint128(1).Lsh(pos).Not()) {
 		pos = uint64(i.TrailingZeros())
 
 		newChange := &allocationchange.CopyFileChange{
 			DestPath:   req.destPath,
+			Uuid:       uid,
```

Comment:
The uid is empty, should we generate one for each, or just use the empty one?

Reply:
No it should not be empty.